fitbenchmarking/results_processing/fitting_report.py

```diff
@@ -26,6 +26,8 @@ def create(results, support_pages_dir, options):
     """
 
     for prob_result in results:
+        if prob_result.accuracy == float('inf'):
```

Comment:
Here too, although I guess this would require an import.
```suggestion
        if np.isinf(prob_result.accuracy):
```